components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/model/UserRecoveryFlowData.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.recovery.model;
+
+import java.sql.Timestamp;
+
+/**
+ * This object represents an entry of the identity metadata database.
+ */
+public class UserRecoveryFlowData {
```

Comment:
What is the requirement for this class? this looks similar to the User Recovery Data class above.

Reply:
This does not include user details similar to User Recovery Data class. It contains the recovery flow details (recovery flow Id, failed attempts, resend count and recovery flow id created time) related to a specific recovery flow id.